commit 8702648befb0a5d7aad111530f20516e3fbc20b9 (refs/changes/98/3298/5)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-02-14T16:39:01-08:00 (1 year, 8 months ago)
    
    OS-6585 Update mdb_v8
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/mdb_v8/Makefile b/mdb_v8/Makefile
index 6c57dfd..92eb96f 100644
--- a/mdb_v8/Makefile
+++ b/mdb_v8/Makefile
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright (c) 2017, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -23,7 +23,7 @@
 # 3. From one directory up, run tar cvzf mdb_v8-<ver>.tar.gz mdb_v8-<ver>
 #
 
-VER =	mdb_v8-1.1.4
+VER =	mdb_v8-1.3.0
 
 include ../Makefile.defs
 include ../Makefile.targ
diff --git a/mdb_v8/mdb_v8-1.1.4.tar.gz b/mdb_v8/mdb_v8-1.1.4.tar.gz
deleted file mode 100644
index 5554cc9..0000000
Binary files a/mdb_v8/mdb_v8-1.1.4.tar.gz and /dev/null differ
diff --git a/mdb_v8/mdb_v8-1.3.0.tar.gz b/mdb_v8/mdb_v8-1.3.0.tar.gz
new file mode 100644
index 0000000..184f612
Binary files /dev/null and b/mdb_v8/mdb_v8-1.3.0.tar.gz differ
